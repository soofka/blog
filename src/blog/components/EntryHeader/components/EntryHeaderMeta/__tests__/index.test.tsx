import { shallow } from 'enzyme';
import * as React from 'react';

import { EntryHeaderMeta } from '../';

describe('<EntryHeaderMeta/>', () => {

  it('renders correctly', () => {
    expect(shallow(
      <EntryHeaderMeta
        created="testCreated"
        updated="testUpdated"
      />,
    )).toMatchSnapshot();
  });

});
