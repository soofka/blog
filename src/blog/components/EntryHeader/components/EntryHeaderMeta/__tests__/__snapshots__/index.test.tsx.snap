// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntryHeaderMeta/> renders correctly 1`] = `
<styled.p>
  <inject-_class-with-languageStore
    name="entry__created"
  />
  <span>
    testCreated
  </span>
  <inject-_class-with-languageStore
    name="entry__last_updated"
  />
  <span>
    testUpdated
  </span>
</styled.p>
`;
